build/Dockerfile.amd64

```diff
@@ -22,9 +29,8 @@ LABEL name="Calico Operator" \
       description="Calico Operator manages the lifecycle of a Calico or Tigera Secure installation on Kubernetes or OpenShift" \
       maintainer="Laurence Man <laurence@tigera.io>"
 
-# Add in top-level license file
-RUN mkdir /licenses
-COPY LICENSE /licenses
+COPY --from=ubi /etc/passwd /etc/passwd
```

Comment:
Do you know who uses this file? I thought it was used only as a part of entrypoint script and perhaps now it's not required?

Reply:
It does not appear to be needed. I can remove if that's what we want.

Reply:
Not needed. I can remove it if that's what we want.

Reply:
If it's not strictly required I am okay removing it. I would like @tmjd's opinion as well.

Reply:
Just FYI, removing following I was able to start from scratch and all worked (tested with crc).
 
-COPY --from=ubi /etc/passwd /etc/passwd
-COPY --from=ubi /licenses /licenses

Reply:
I think `-COPY --from=ubi /licenses /licenses` might be required but probably not the copy for `/etc/passwd`.

Reply:
I don't know of a reason it is needed. If it works without I'm happy for it to be removed.
Awesome @gianlucam76, thanks for doing this.